integration_test/src/event_stream_reader_tests.rs

```diff
@@ -28,11 +32,136 @@ pub fn test_event_stream_reader(config: PravegaStandaloneServiceConfig) {
     let client_factory = ClientFactory::new(config);
     let handle = client_factory.get_runtime_handle();
     handle.block_on(test_read_api(&client_factory));
-    handle.block_on(test_stream_scaling(&client_factory))
+    handle.block_on(test_stream_scaling(&client_factory));
+    handle.block_on(test_release_segment(&client_factory));
+    handle.block_on(test_release_segment_at(&client_factory));
+    test_multiple_readers(&client_factory);
+    test_reader_offline(&client_factory);
 }
 
-async fn test_stream_scaling(client_factory: &ClientFactory) {
+async fn test_release_segment(client_factory: &ClientFactory) {
     let scope_name = Scope::from("testReaderScaling".to_owned());
+    let stream_name = Stream::from("testReaderRelease1".to_owned());
+
+    const NUM_EVENTS: usize = 10;
+
+    let new_stream = create_scope_stream(
+        client_factory.get_controller_client(),
+        &scope_name,
+        &stream_name,
+        1,
+    )
+    .await;
+    // write events only if the stream is created. This is useful if we are running the reader tests
+    // multiple times.
+    if new_stream {
+        write_events_before_and_after_scale(
+            scope_name.clone(),
+            stream_name.clone(),
+            &client_factory,
+            NUM_EVENTS,
+        )
+        .await;
+    }
+    let str = ScopedStream {
```

Comment:
I think this is a reserved key word

Reply:
yeah but the compiler never threw an error though. I have renamed this variable.